packages/components/src/ObjectViewer/ObjectViewer.component.js

```diff
@@ -1,15 +1,21 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import Table from './Table';
+import GenericViewer from './Generic';
 import JSONLike from './JSONLike';
 import List from './List';
+import Model from './Model';
+import Records from './Records';
+import Table from './Table';
 
 export const DISPLAY_MODES = {
 	FLAT: 'flat',
+	GENERIC: 'generic',
+	LIST: 'list',
+	MODEL: 'model',
+	RECORDS: 'records',
 	TABLE: 'table',
 	TREE: 'tree',
```

Comment:
What's the difference between the `records` and `tree` view?

Reply:
Records display a list of record and each record is displayed with a viewer based on the generic one.
Generic includes quality indicator, that is used in model viewer too.
The idea behind the genericViewer is that it should be the common part of all these type of viewers, and allow each specific viewer to be based on it with some customisation. In a perfect world, tree and object would be refactored to use GenericViewer